src/libraries/System.Text.Json/codegenerator.Tests/JsonSerializerSourceGeneratorTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
```

Comment:
We need a license header on all `.cs` files, e.g https://github.com/dotnet/runtime/blob/b93d5707caa7421ca4696746384b6a14c3e28d63/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonConverterAttribute.cs#L1-L3.